packages/app/src/specs/generators/component/ComponentGeneratorStepOne.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <div class="flex flex-col flex-grow justify-between">
+    <template v-if="generatedSpecError">
+      <EmptyGenerator
+        :gql="generateSpecFromSource.currentProject"
+        title=""
+        type="component"
+        :other-generators="false"
+        :spec-file-name="generatedSpecError.fileName"
+        :errored-codegen-candidate="generatedSpecError.erroredCodegenCandidate"
+        :has-custom-spec-pattern="hasCustomSpecPattern"
+        @restart="cancelSpecNameCreation"
+        @updateTitle="(value) => emits('update:title', value)"
+      />
+    </template>
+
+    <template v-else>
+      <div class="flex-grow">
+        <div
+          v-if="mutation.fetching.value"
+          class="mt-48px w-full inline-flex items-center justify-center"
+        >
+          <i-cy-loading_x16 class="h-48px mr-12px animate-spin w-48px" />
+          <p class="text-lg">
+            Loading
+          </p>
+        </div>
+        <FileChooser
+          v-else-if="!result"
+          v-model:extensionPattern="extensionPattern"
+          :files="allFiles"
+          :loading="query.fetching.value"
+          :file-match-button-disabled="true"
```

Comment:
In this case, we want the file match button to be disabled so that the user can't change the glob pattern.